src/components/Globe.jsx

```diff
@@ -1,84 +1,115 @@
-import React, { useState, useCallback, useEffect } from 'react';
-import ReactGlobe from 'react-globe';
-import Responsive from 'react-responsive-decorator';
+import React, { useState, useCallback, useEffect, Suspense } from 'react';
 import axios from 'axios';
-import StickyHeadTable from './Table';
+// import { render } from '@testing-library/react';
+import Responsive from 'react-responsive-decorator';
 import Header from './Header';
+import Preloader from './Preloader';
 import ZoomButton from './ZoomButton';
-import globeTexture from '../images/globe_bg.png';
+// import Grid from '@material-ui/core/Grid';
+// import Typography from '@material-ui/core/Typography';
+import globeTextureImage from '../images/background.jpg';
 
+import './style.css';
 import 'tippy.js/dist/tippy.css';
 import 'tippy.js/animations/scale.css';
 
 
-const colorScale = (code) => {
+// Lazy Loading React COmponent
+const ReactGlobe = React.lazy(() => import('react-globe.gl'));
+const StickyHeadTable = React.lazy(() => import('./Table'));
+
+// Context Manager
+export const DateContext = React.createContext(`${new Date().toISOString().slice(0, 10)}`)
```

Comment:
Since context is globally available in the application. We shall define it on the top level, e.g: `App.jsx` so that it's available to other components.